### DISCUSSION TITLE
Temperature Sweep Examples

### DISCUSSION THREAD

**Discussion by tomhajjar** [ra3xdh/qucs_s#935] [Show and tell]
Per #925, I made three examples using Temperature Sweep of a "Blue" BJT. 2 using ngspice and 1 using qucsator-rf.

![2024-09-06_203319](https://github.com/user-attachments/assets/d0e94453-7333-48bb-8777-8bff7752df5f)
![2024-09-06_203233](https://github.com/user-attachments/assets/e8f59ac9-9d8e-4b90-b2c8-81832e69b3a7)
![2024-09-06_203300](https://github.com/user-attachments/assets/bed37a29-b39c-4178-90f0-2205d7d061e5)

[BJT_Temp_Sweep.zip](https://github.com/user-attachments/files/16920435/BJT_Temp_Sweep.zip)

**Comment by ra3xdh** [comment]
Thanks! I have uploaded the examples into the repository.
